keyboards/treasure/type9s3/readme.md

```diff
@@ -0,0 +1,21 @@
+# Treasure TYPE-9 Series III
+
+![TYPE9SIII](https://i.imgur.com/lohJ0Dal.png)
```

Comment:
```suggestion
![TYPE9SIII](https://i.imgur.com/lohJ0Dalh.png)
```

Reply:
what is the difference between lohJ0Dal.png and lohJ0Dalh.png ?

Reply:
> what is the difference between lohJ0Dal.png and lohJ0Dalh.png ?

[quick explanation](https://thomas.vanhoutte.be/miniblog/imgur-thumbnail-trick/)